akka-actor/src/main/scala-2.13+/akka/util/ccompat/package.scala

```diff
@@ -14,4 +14,10 @@ package akka.util
 package object ccompat {
   private[akka] type Factory[-A, +C] = scala.collection.Factory[A, C]
   private[akka] val Factory = scala.collection.Factory
+
+  // When we drop support for 2.12 we can delete this concept
+  // and import scala.jdk.CollectionConverters.Ops._ instead
```

Comment:
Maybe add a new issue with a GH issue number so the task doesn't get lost in the ether.

Reply:
https://github.com/akka/akka/issues/27008